src/main/java/net/mcreator/workspace/elements/FolderElement.java

```diff
@@ -58,10 +58,11 @@ public void addChild(FolderElement child) {
 			}
 
 			// we will not add the same child again
-			return;
+			return child;

```

Comment:
I forgot it in my last review, but you can't return a FolderElement if the return type of the method is `void`.